README.md

```diff
@@ -12,96 +12,96 @@ Pull requests are welcome. Please keep the list in alphabetical order.
-[Toggl](https://www.toggl.com/)                   | Time Tracking Tool                       | Ruby, C#, Go, Mobile, Web                      | ✗                                    | ?                       | :keyboard: [Jobs](https://jobs.toggl.com/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Toggl-EI_IE734060.11,16.htm)
-[Toptal](https://www.toptal.com/)                 | Tech freelancer marketplace              | Ruby, Web                                      | ✗                                    | ?                       | :muscle: [Core team positions](https://www.toptal.com/careers)<br>:computer: [Apply as a developer](https://www.toptal.com/developers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Toptal-EI_IE882070.11,17.htm)
-[Trello](https://trello.com)                      | Trello                                   | Node, Mobile, Web                              | ✓                                    | ?                       | :keyboard: [Jobs](https://trello.com/jobs)<br>:door: [Glassdoor (acquired by Atlassian)](https://www.glassdoor.com.au/Overview/Working-at-Atlassian-EI_IE115699.11,20.htm)
-[Tyk](https://tyk.io/)                            | Cloud native API Management, open source | Go, Web, DevOps                                | ✓                                    | ?                       | :keyboard: [Jobs](https://tyk.io/about/team/)<br>:door: [Glassdoor ](https://www.glassdoor.co.uk/Reviews/Tyk-Reviews-E2321465.htm)
-[Universal Mind](http://www.universalmind.com/)   | Digital agency                           | Mobile, Web                                    | ✓                                    | ?                       | :keyboard: [Jobs](http://www.universalmind.com/aboutus)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Universal-Mind-EI_IE648227.11,25.htm)
+[Stack Exchange](http://stackexchange.com/)       | Stack Overflow & more                    | .NET, Web                                      | ✓                                    | US / EU                 | :keyboard: [Jobs](http://stackoverflow.com/company/work-here)<br>:mortar_board: [Philosophy](http://blog.stackoverflow.com/2013/02/why-we-still-believe-in-working-remotely/)<br>:moneybag: [Salary calculator](https://stackoverflow.com/company/salary/calculator)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Stack-Overflow-EI_IE504626.11,25.htm)
+[SUSE](https://www.suse.com/)                     | SUSE Linux distribution                  | Many                                           | ?                                    | No remote jobs          | :keyboard: [Jobs](https://jobs.suse.com/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-SUSE-EI_IE466462.11,15.htm)
```

Comment:
Maybe leave as a question mark and open a separate PR to remove?

Reply:
:+1